Gemfile

```diff
@@ -14,6 +14,13 @@ gem 'capybara'
 %w[rspec-core rspec-expectations rspec-mocks rspec-rails rspec-support].each do |lib|
   gem lib, :git => "https://github.com/rspec/#{lib}.git", :branch => 'master'
 end
+# Factory Girl - Testing
+gem "factory_girl_rails"
+gem "factory_girl"
```

Comment:
You only need "factory_girl_rails" because factory_girl_rails automatically incorporates the factory_girl gem and adds support for Rails